docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -0,0 +1,126 @@
+========================
+OAuth 2.0 authentication
+========================
+
+Trino can be configured to enable OAuth 2.0 authentication over HTTPS for the
+Web UI and the JDBC driver. Trino use `Authorization Code
```

Comment:
why only the Web UI and the JDBC driver..  what about the CLI and any other clients?

Reply:
Maybe

Trino uses the Authorization Code flow